activities/tool-management/1password.md

```diff
@@ -0,0 +1,26 @@
+---
+type: resource
+---
+
+# 1Password
+
+1Password is a password manager developed by AgileBits Inc. It provides a place for users to store various passwords, software licenses, and other sensitive information in a virtual vault that is locked with a PBKDF2-guarded master password.
+
+## Using 1Password
+
+* Jitsi Meet can be reached on 'publiccodenet.1password.com'.
+* To ensure security and consistency, each employee has to use 1Password.
+* If log-in credentials needs to be shared, it has to be done within 1Password using the "send a copy" option. It's not allowed to share passwords in plaintext (email).
+* A strong Master Password needs to be set to open the vault.
+* There is also a Secret Key which is created locally on the user's device. It’s combined with the Master Password to authenticate the user with the 1Password server and to encrypt your 1Password data.
+* 1Password [has extensive documentation](https://support.1password.com/)
+
+## 1Password management
+
+All team members receive an account from the Operations Coordinator during onboarding. The Operations Coordinator and Chief Executive have administrative access to 1Password. They can kickstart recovery processes in case an employee loses access to 1Password, as well as set up vaults for sharing and assign access rights to existing vaults.
+
+## Privacy and GDPR
+
+The services offered by AgileBits, Inc. through 1Password fully comply with the requirements of the European Union’s General Data Protection Regulation (GDPR). Data by or about any individual covered by the GDPR may be managed using these services. Read more about [AgileBits Privacy Policy here.](https://1password.com/legal/privacy/)
```

Comment:
```suggestion
The services offered by AgileBits, Inc. through 1Password fully comply with the requirements of the European Union’s General Data Protection Regulation (GDPR). European data is held on servers located in Europe. Read more about [AgileBits Privacy Policy here.](https://1password.com/legal/privacy/)
```

Reply:
It's more helpful to tell people how the data complies with GDPR than to repeat a company's assertion that it does.

Reply:
I've made this change myself now.